public/scripts/extensions/quick-reply/src/QuickReplySet.js

```diff
@@ -239,6 +239,28 @@ export class QuickReplySet {
         this.save();
         return qr;
     }
+    addQuickReplyFromText(qrJson) {
+        let data;
+        try {
+            data = JSON.parse(qrJson ?? '{}');
+            delete data.id;
+        } catch {
+            // not JSON data
```

Comment:
Not sure if I wouldn't at least still log this to console as an info or error.
Sure, most of the time it might not be JSON, but it could also be malformatted JSON. And to be able to see the reason, having the json parser error with line number might be helpful.
But that's nitpicking again.